<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="0" tests="1" failures="1">
  <testsuite name="Root Suite" timestamp="2020-06-04T20:44:05" tests="0" file="cypress/integration/apps/service_types/ST07.js" failures="0" time="0">
  </testsuite>
  <testsuite name="7 - I can access Service Type Details page by clicking on the name in Inspector Panel" timestamp="2020-06-04T20:44:05" tests="1" failures="1" time="0">
    <testcase name="7 - I can access Service Type Details page by clicking on the name in Inspector Panel ST-7-1 Should access details page from Inspector header" time="0" classname="ST-7-1 Should access details page from Inspector header">
      <failure message="Timed out retrying: cy.click() failed because this element is detached from the DOM.

&lt;div role=&quot;row&quot; row-index=&quot;0&quot; aria-rowindex=&quot;2&quot; row-id=&quot;0&quot; comp-id=&quot;73&quot; class=&quot;ag-row ag-row-no-focus ag-row-even ag-row-level-0 ag-row-position-absolute ag-row-first ag-row-last&quot; style=&quot;height: 36px; transform: translateY(0px); border: none;&quot;&gt;&lt;/div&gt;

Cypress requires elements be attached in the DOM to interact with them.

The previous command that ran was:

  &gt; cy.contains()

This DOM element likely became detached somewhere between the previous and current command.

Common situations why this happens:
  - Your JS framework re-rendered asynchronously
  - Your app code reacted to an event firing and removed the element

You typically need to re-query for the element or add &apos;guards&apos; which delay Cypress from running new commands.

https://on.cypress.io/element-has-detached-from-dom" type="CypressError"><![CDATA[CypressError: Timed out retrying: cy.click() failed because this element is detached from the DOM.

<div role="row" row-index="0" aria-rowindex="2" row-id="0" comp-id="73" class="ag-row ag-row-no-focus ag-row-even ag-row-level-0 ag-row-position-absolute ag-row-first ag-row-last" style="height: 36px; transform: translateY(0px); border: none;"></div>

Cypress requires elements be attached in the DOM to interact with them.

The previous command that ran was:

  > cy.contains()

This DOM element likely became detached somewhere between the previous and current command.

Common situations why this happens:
  - Your JS framework re-rendered asynchronously
  - Your app code reacted to an event firing and removed the element

You typically need to re-query for the element or add 'guards' which delay Cypress from running new commands.

https://on.cypress.io/element-has-detached-from-dom
    at cypressErr (https://chris231.test.l7informatics.com/__cypress/runner/cypress_runner.js:138036:9)
    at throwErr (https://chris231.test.l7informatics.com/__cypress/runner/cypress_runner.js:137969:11)
    at Object.throwErrByPath (https://chris231.test.l7informatics.com/__cypress/runner/cypress_runner.js:138017:3)
    at Object.retry (https://chris231.test.l7informatics.com/__cypress/runner/cypress_runner.js:132299:19)
    at retryActionability (https://chris231.test.l7informatics.com/__cypress/runner/cypress_runner.js:120118:19)
    at tryCatcher (https://chris231.test.l7informatics.com/__cypress/runner/cypress_runner.js:164895:23)
    at Function.Promise.attempt.Promise.try (https://chris231.test.l7informatics.com/__cypress/runner/cypress_runner.js:162170:29)
    at tryFn (https://chris231.test.l7informatics.com/__cypress/runner/cypress_runner.js:132763:21)
    at whenStable (https://chris231.test.l7informatics.com/__cypress/runner/cypress_runner.js:132798:12)
    at https://chris231.test.l7informatics.com/__cypress/runner/cypress_runner.js:132342:16
    at tryCatcher (https://chris231.test.l7informatics.com/__cypress/runner/cypress_runner.js:164895:23)
    at Promise._settlePromiseFromHandler (https://chris231.test.l7informatics.com/__cypress/runner/cypress_runner.js:162831:31)
    at Promise._settlePromise (https://chris231.test.l7informatics.com/__cypress/runner/cypress_runner.js:162888:18)
    at Promise._settlePromise0 (https://chris231.test.l7informatics.com/__cypress/runner/cypress_runner.js:162933:10)
    at Promise._settlePromises (https://chris231.test.l7informatics.com/__cypress/runner/cypress_runner.js:163012:18)
    at Promise._fulfill (https://chris231.test.l7informatics.com/__cypress/runner/cypress_runner.js:162957:18)]]></failure>
    </testcase>
  </testsuite>
</testsuites>